simulation/modules/fungus.py

```diff
@@ -75,230 +240,74 @@ def cell_list_factory(self: 'FungusState'):
 @attr.s(kw_only=True)
 class FungusState(ModuleState):
     cells: FungusCellList = attr.ib(default=attr.Factory(cell_list_factory, takes_self=True))
-    init_num: int = 0
-    p_lodge: float = 0.1
-    p_internal_swell: float = 0.05
-    ITER_TO_CHANGE_STATUS: int = 1
-    iron_min: int = 0
-    iron_max: float = 0.0
-    iron_absorb: float = 0.0
-    spacing: float = 0.0
-    iron_min_grow: float = 0.0
-    grow_time: float = 0.0
-    p_branch: float = 0.0
-    p_internalize: float = 0.0
-    health: float
```

Comment:
I think some of the other modules read this value because they do damage (dec fungus health) according to a percent of total initial health. Please correct me if I am wrong, but not including it also means the health is not configurable by the user.

Reply:
I think `health` is to define the max health of a spore right? If other modules damage fungus health, they will change state.fungus.cells['health'] but not the health configuration.

Reply:
Yes that is correct but the current way the other modules calculate how much to alter state.fungus.cells['health'] is based on "health". For example, consider the epithelial cell. In matt's model, the function for damage_conidia by epithelial cells is:

health_f = health_f − (init health(f) · time step/e_kill)

in this case state.fungus.cells['health'] == health_f and "health" == init_health(f)
The units of the kill constant e_kill which is a configurable parameter is hours. So if e_kill = 30 and if the time step is 24hrs then the epithelial cell does 0.8 * "health" damage each time step.

Reply:
I see. So health is damaged by a proportion.

Reply:
Yes, but a proportion of the initial health not current health